apps/src/appMain.js

```diff
@@ -52,7 +52,8 @@ module.exports = function(app, levels, options) {
   if (studioApp.isUsingBlockly()) {
     var blockInstallOptions = {
       skin: options.skin,
-      isK1: options.level && options.level.isK1
+      isK1: options.level && options.level.isK1,
+      level: options.level
```

Comment:
Passing the whole level object to our block installer seems a little overkill. Maybe we should just extract the properties we need and pass those along (like we do for isK1). (Note: this feedback doesnt need to block merge)